tokio/src/runtime/handle.rs

```diff
@@ -1,47 +1,41 @@
+#[cfg(feature = "rt-core")]
 use crate::runtime::current_thread;
 #[cfg(feature = "rt-full")]
 use crate::runtime::thread_pool;
-use crate::runtime::JoinHandle;
+use crate::runtime::{blocking, io, time};
+#[cfg(feature = "rt-core")]
+use crate::task::JoinHandle;
 
+#[cfg(feature = "rt-core")]
 use std::future::Future;
 
-/// Spawns futures on the runtime
-///
-/// All futures spawned using this executor will be submitted to the associated
-/// Runtime's executor. This executor is usually a thread pool.
-///
-/// For more details, see the [module level](index.html) documentation.
+/// Handle to the runtime
 #[derive(Debug, Clone)]
-pub struct Spawner {
-    kind: Kind,
+pub struct Handle {
```

Comment:
Fields are `pub(super)` as the type is constructed from `builder.rs`.

Reply:
what's the rationale for renaming `Spawner` to `Handle`? `Spawner` is a bit more descriptive IMO, and I think past `tokio` (esp. `tokio-core`)'s overuse of the name `Handle` was unclear to some users...

Reply:
It does more than spawning.